invokeai/app/services/item_storage/item_storage_sqlite.py

```diff
@@ -13,13 +10,6 @@
 
 
 class SqliteItemStorage(ItemStorageABC, Generic[T]):
```

Comment:
Can we delete this class altogether?

Seems like it's unused at this point. And, I think it might be the last remaining piece of code that changes the SQL schema outside of the migrator, so I don't want anyone to be tempted to revive this as-is.